src/test/java/com/loadero/models/TestParticipant.java

```diff
@@ -81,6 +81,8 @@ public void testCreateAndDeleteParticipant() throws IOException {
             .build();
         com.loadero.model.Participant create = com.loadero.model.Participant.create(params);
         assertNotNull(create);
+        System.out.println(Location.FRANKFURT);
```

Comment:
?

Reply:
Wanted to see what it prints out. And it's actually prints out AWS instance's name.

Reply:
shouldn't they be removed then?

Reply:
Yeah, I guess